framework/tests/src/tests/RangeTest.java

```diff
@@ -183,6 +183,7 @@ public void remove() {
 
         public ValuesInRangeIterator(Range range) {
             this.range = range;
+            Range.IGNORE_OVERFLOW = false;
```

Comment:
How is this change related to the rest of the pull request?

Reply:
This pull request uncovered a bug in the Value Checker.  I'll move the fix to another branch.

Reply:
Is this something that should be pulled into this PR?